packages/editor/src/components/EditorContainer.tsx

```diff
@@ -184,11 +184,12 @@ const onSaveAs = async () => {
   try {
     if (sceneName || editorState.sceneModified.value) {
       const blob = await takeScreenshot(512, 320, 'ktx2')
+      const thumbnailBlob = await takeScreenshot(256, 160, 'jpeg')
```

Comment:
we dont want to be taking two screenshots, we are able to display ktx2s as images